components/style/properties/data.py

```diff
@@ -220,12 +230,19 @@ def new_style_struct(self, *args, **kwargs):
     def active_style_structs(self):
         return [s for s in self.style_structs if s.additional_methods or s.longhands]
 
+    def add_prefixed_aliases(self, property_data):
+        # FIXME Servo's DOM architecture doesn't support vendor-prefixed properties.
```

Comment:
FWIW Gecko has a bug here where CSSSD can be indexed with `webkit-foo` without the preceding dash.